components/expenses/PayExpenseModal.js

```diff
@@ -79,9 +79,14 @@ const getPayoutLabel = (intl, type) => {
   return i18nPayoutMethodType(intl, type, { aliasBankAccountToTransferWise: true });
 };
 
-const getPayoutOptionValue = (payoutMethodType, isAuto, host) => {
+const getPayoutOptionValue = (payoutMethod, isAuto, host) => {
+  const payoutMethodType = payoutMethod?.type || PayoutMethodType.OTHER;
```

Comment:
Forcing "OTHER" here prevented from selecting back "Automatic" in the switcher. Pushing a fix in https://github.com/opencollective/opencollective-frontend/pull/9667/files#diff-46d5dbc2ddcf09107ae2f24a445877429a4d69d990528353034ab11a53746a39R83.